### DISCUSSION TITLE
Engine description

### DISCUSSION THREAD

**Discussion by RolandTi** [Sikarugir-App/Sikarugir#24] [General]
Hello,
maybe I'm missing a wiki or something in the Kegworks-App Github,
but maybe the engine list could add a hint for each engine ?

### Crossover Wine ?
WS12WineCX23.7.1
WS11WineCX21.2.0
WS11WineCX20.0.4
WS11WineCX19.0.2

### Apple Wine ?
WS12WineGPTK1.1

### Raw wine from Whisky App team ?
WS12WhiskyWine2.4.4

### Crossover Wine for 32 bits app ?
WS11WineCX32Bit20.0.4
WS11WineCX32Bit21.2.0
WS11WineCX32Bit19.0.2

Thanks !
Have a nice day

**Comment by Gcenx** [comment]
WineCX engines are based on crossover-sources

WineGPTK is Apples Game PortingToolkit see [source](https://github.com/Kegworks-App/wine/tree/Game-Porting-Toolkit-1.1-Fixup)

WhiskyWine is the version listed with homebrew dylibs removed.

Engines marked 32Bit _only_ support 32Bit software, there’s a couple of weird edge cases that require them.

<br>

Currently only WineCX23.7.1 is supported, other Engines are being provided for possible edge cases.

**Reply by RolandTi** [reply]
Perfect, thanks !
